src/utils/queries.js

```diff
@@ -7,4 +7,13 @@ const getResource = async (_key, id) => {
   return data;
 };
 
-export { getResource };
+const getResources = async searchTerm => {
```

Comment:
It would be nice to have a test for this, mainly for documentation or what we expect to get in return.

Reply:
👍 How does this look? 7845982